apps/nextra/pages/en/build/smart-contracts/object/using-objects.mdx

```diff
@@ -205,47 +205,55 @@ Generated event handles can be transferred to the Object as long as you have the
 ```move filename="example.move"
 module 0x42::example {
   use aptos_framework::event;
-  use aptos_framework::fungible_asset::FungibleAsset;
+  use aptos_framework::fungible_asset::Metadata;
   use aptos_framework::object::{Self, Object};
 
```

Comment:
Looks like this is also missing an object resource group here

Reply:
Added it. Is this fine?

Reply:
It's perfect thanks!

Reply:
Glad you liked it! Happy to contribute :)